config/competitions_config.yml.example

```diff
@@ -0,0 +1,55 @@
+development: # The following should be defined for every environment.
+  default_url_options: { host: , # e.g. config.app_domain
+                         port:   # e.g. 3000
+                       }
+  subdomain: # e.g. ''
+  app_domain: # e.g. 'localhost'
+  database: # https://guides.rubyonrails.org/configuring.html#configuring-a-database
+    adapter: # e.g. postgres
+    host: # e.g. localhost
+    database: # <DATABASE_NAME>
+    username: # <DATABASE_USERNAME>
+  aws: # https://edgeguides.rubyonrails.org/active_storage_overview.html#amazon-s3-service
```

Comment:
Should there be different storage options here? If so, open a new ticket to update this file rather than block this PR. Close or resolve this comment as needed.